src/architecture/utilities/tests/test_rigidBodyKinematicsC.cpp

```diff
@@ -0,0 +1,1134 @@
+/*
```

Comment:
Why is this one named with a "C" suffix? Why not just `test_rigidBodyKinematics.cpp`?